src/ontology/cl-edit.owl

```diff
@@ -31685,6 +31688,50 @@ AnnotationAssertion(rdfs:label obo:CL_4033056 "onychocyte")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033058 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033058 <https://orcid.org/0000-0003-4969-670X>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033058 "2023-10-16T08:30:25Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://nrs.harvard.edu/URN-3:HUL.INSTREPOS:37368328") oboInOwl:hasBroadSynonym obo:CL_4033058 "ER-positive luminal hormone-sensor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2023.04.21.537845") oboInOwl:hasBroadSynonym obo:CL_4033058 "hormone sensing cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") oboInOwl:hasBroadSynonym obo:CL_4033058 "hormone-sensing mammary epithelial cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37380767") oboInOwl:hasBroadSynonym obo:CL_4033058 "luminal hormone-responsive cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20346151") oboInOwl:hasBroadSynonym obo:CL_4033058 "luminal mature cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2023.04.21.537845") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "HS")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "HS-MEC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-6677-8489") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "LHS")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37380767") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "LumHR")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") Annotation(oboInOwl:hasDbXref "PMID:34156490") Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0003-4969-670X") rdfs:comment obo:CL_4033058 "Following lineage specification, a luminal hormone-sensing cell expresses receptors for one or more hormones, such as estrogens, progesterone, androgens, and prolactin.
+In response to progesterone, luminal hormone-sensing cells express TNFSF11, WNT4, and RSPO1, among other factors, which cause branching, proliferation, and partial differentiation in LASP and B-Myo cells during luteal phases of the estrous/menstrual cycle and during gestation.
+During ductal elongation, ER activity drives expression of AREG in LHS cells, which is necessary for proliferation of the epithelium to fill the mammary fat pad.")
+AnnotationAssertion(rdfs:label obo:CL_4033058 "luminal hormone sensing cell")
+SubClassOf(obo:CL_4033058 obo:CL_0000512)
+SubClassOf(obo:CL_4033058 obo:CL_0002326)
+SubClassOf(obo:CL_4033058 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000007204))
```

Comment:
Don't see this reflected in text def.

Reply:
Asking to SME about if all LHS cells have estrogen receptors:
> LHS cells are typically ESR1/ERa-high (they do NOT express ERb), especially in younger women and mice, when hormone levels are high. In older women, ER levels can drop off in at least some cells, but it is a pretty robust marker generally speaking. In our experience, FOXA1 (a master TF of this lineage and pioneer factor for ER) may be slightly more reliable than ER. 

As I see it, even if the levels are lower at certain points, if it is a robust marker it should be included. I will add it to the textual definition.